contracts/aggregator/mainnet/flashloan/advancedRoutesImpl/helpers.sol

```diff
@@ -0,0 +1,226 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "./variables.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract Helper is ConstantVariables {
+    using SafeERC20 for IERC20;
+
+    function calculateAndReadFromStorage(address key_) public view returns (address result_) {
+        bytes32 slot_ = keccak256(abi.encode(key_, 2)); // Slot for `tokenToCToken` variable
+
+        assembly {
+            result_ := sload(slot_) // read value from the storage slot
```

Comment:
@shriyatyagii This is for tokenToCToken[_tokens[i]] replacement, right?

Reply:
You can use this as well, why not just add `tokenToCToken` in the same slot on variables file?

Reply:
Yes, we could've added. But just wanted to avoid anything related to updating the storage slots on implementation.